src/train/train_model.py

```diff
@@ -6,6 +6,8 @@
 from corpy.udpipe import pprint
 from typing import List
 import re
+import argparse
```

Comment:
Is this module an installed dependency?

Reply:
Yes, argparse is an inner model of Python